keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -620,7 +620,11 @@ main focus will be on kubectl and golang clients.
 
 #### GA
 
-- TBD
```

Comment:
A few nits to earlier parts of this doc:
1. Make sure to update the `Release Signoff Checklist` as appropriate. 
2. In the `Test Plan` section please make sure to update appropriate sections with links to testgrid showing passing tests and listing current test coverage.

Reply:
Updated